src/Components/Grants/GrantShowOverview.jsx

```diff
@@ -1,29 +1,140 @@
-/* eslint-disable no-unused-vars */
-import React, { useState } from "react";
+import React, { useEffect } from "react";
 import { useQuery, useMutation } from "react-query";
 import { useParams } from "react-router-dom";
+import Button from "../design/Button/Button";
 import Container from "../design/Container/Container";
-import SectionListItem from "../Sections/SectionListItem";
+import Hero from "../design/Hero/Hero";
+import { useCurrentOrganization } from "../../Contexts/currentOrganizationContext";
+import * as GrantsService from "../../Services/Organizations/GrantsService";
+import * as SectionsService from "../../Services/Organizations/Grants/SectionsService";
+import countSectionWords from "../../Helpers/countSectionWords";
+import { SortableItem } from "./SortableItem";
+import { Item } from "./Item";
+import CurrentOrganizationLink from "../Helpers/CurrentOrganizationLink";
+import {
+  SortableContext,
+  verticalListSortingStrategy,
+} from "@dnd-kit/sortable";
+import { DragOverlay } from "@dnd-kit/core";
+
 import "./GrantShowOverview.css";
-import SortableElement from "../Elements/SortableElement";
 
-export default function GrantShowOverview({ grant }) {
+function countTotalSectionsWords(sections = []) {
+  return sections?.reduce(
+    (total, section) => total + countSectionWords(section),
+    0
+  );
+}
+
+export default function GrantShowOverview(props) {
+  const { currentOrganization, organizationClient } = useCurrentOrganization();
+  const { grantId } = useParams();
+
+  const {
+    data: grant,
+    isError,
+    isLoading,
+    error,
+  } = useQuery("getGrant", () =>
+    GrantsService.getGrant(organizationClient, grantId)
+  );
+  const totalWordCount = countTotalSectionsWords(grant?.sections);
+
+  const grantSectionsReorder = () => {
+    const sectionsToReorder = [];
+    props.sortableSections.forEach((newSection, index) => {
+      const checkSection = grant.sections[index];
+      if (newSection.sortOrder !== checkSection.sortOrder) {
+        sectionsToReorder.push({
+          id: newSection.id,
+          sort_order: index,
+        });
+      }
+    });
+    if (sectionsToReorder.length > 0) {
+      reorderSections(sectionsToReorder);
+    }
+  };
+
+  const { mutate: reorderSections } = useMutation(
+    (sectionsToReorder) =>
+      SectionsService.reorderSections(
+        organizationClient,
+        grantId,
+        sectionsToReorder
+      ),
+    {
+      onSuccess: () => {
+        alert("Sections reordered!");
+      },
+    }
+  );
+
+  useEffect(() => {
+    props.setSortableSections(grant.sections);
+  }, []);
+
+  if (isLoading) {
+    return <span>Loading...</span>;
+  }
+
+  if (isError) {
+    return <span>Error: {error.message}</span>;
+  }
+
+  const heroButtons = () => (
+    <>
+      <Button
+        variant="outlined"
+        as={CurrentOrganizationLink}
+        to={`/grants/${grant.id}/`}
+      >
+        Content View
+      </Button>
+    </>
```

Comment:
Since this button has no siblings, we can remove the extra fragment
```suggestion
    <Button
      variant="outlined"
      as={CurrentOrganizationLink}
      to={`/grants/${grant.id}/`}
    >
      Content View
    </Button>
```